atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -541,15 +541,16 @@ def run(self):
 
         koji_metadata, output_files = self.get_metadata()
 
-        try:
-            session = get_koji_session(self.workflow, self.koji_fallback)
-            for output in output_files:
-                if output.file:
-                    self.upload_file(session, output, self.koji_upload_dir)
-        finally:
-            for output in output_files:
-                if output.file:
-                    output.file.close()
+        if not is_scratch_build():
```

Comment:
Is this `if` needed? Why is not here uploading scratch metadata as in exit plugin? (This part is copy pasted, the same code)